app/src/main/res/layout/icon_navigation_item.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/settings_item_icon"
```

Comment:
I think you were using camelCase naming convention for ids. Also it's not settings item icon, it can be any icon. so id should be iconImageView